package.json

```diff
@@ -156,25 +157,31 @@
 		"@wordpress/element": "2.4.0",
 		"@wordpress/i18n": "3.4.0",
 		"@wordpress/jest-preset-default": "4.1.0",
+		"@wordpress/e2e-test-utils": "1.1.0",
+		"@wordpress/eslint-plugin": "2.1.0",
 		"babel-eslint": "10.0.1",
 		"babel-loader": "8.0.6",
 		"chai": "4.2.0",
 		"commander": "2.20.0",
+		"config": "3.1.0",
```

Comment:
Adding this dependency screws up this import:

https://github.com/Automattic/jetpack/blob/597bc7ccbae70888cfa266c2fb30c09c235224cd/_inc/client/lib/analytics/index.js#L10